src/clr-angular/data/datagrid/datagrid-hideable-column.spec.ts

```diff
@@ -13,11 +13,13 @@ import {FiltersProvider} from "./providers/filters";
 import {Page} from "./providers/page";
 import {Sort} from "./providers/sort";
 import {StateDebouncer} from "./providers/state-debouncer.provider";
+import {TableHeightService} from "./providers/table-height.service";
 import {DomAdapter} from "./render/dom-adapter";
 import {DatagridRenderOrganizer} from "./render/render-organizer";
 
-const PROVIDERS_NEEDED =
-    [Sort, FiltersProvider, DatagridRenderOrganizer, DomAdapter, DragDispatcher, Page, StateDebouncer];
+const PROVIDERS_NEEDED = [
```

Comment:
We really need to set this on the test context at the top level of the Datagrid specs. This is getting silly. 😂